pkg/agent/openflow/fields.go

```diff
@@ -82,6 +82,8 @@ var (
 	CustomReasonIGMPRegMark    = binding.NewRegMark(CustomReasonField, CustomReasonIGMP)
 	// reg0[18]: Mark to indicate remote SNAT for Egress.
 	RemoteSNATRegMark = binding.NewOneBitRegMark(0, 18)
+	// reg0[19]: Mark to indicate redirect action of layer 7 NetworkPolicy.
+	L7NPRedirectRegMark = binding.NewOneBitRegMark(0, 19)
```

Comment:
Since the value of this reg will be read, how about defining it as a field, for example:

```go

const DispositionL7NPRedirect  = 0b1

var(
L7NPRegField = binding.NewRegField(0, 19, 19)
L7NPRedirectRegMark = binding.NewRegMark(L7NPRegField, DispositionL7NPRedirect)
)

Reply:
Done.